### PR TITLE
Modal code review

### PR DESCRIPTION
**Pull request by RohitKuwar**
@Kratika0907 Please review the code.

### CODE REVIEW

**Review by Harshi7016 — COMMENTED** [review]
@RohitKuwar there are merge conflicts , please resolve them

**Comment by RohitKuwar** [comment]
@Harshi7016 ok

**Comment by Harshi7016** [comment]
If there is any help needed, do let me know I can join the call

**Comment by RohitKuwar** [comment]
> 
> 
> If there is any help needed, do let me know I can join the call

tysm @Harshi7016

File: components/Modal/index.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
-const Modal = () => {
-  return <div> </div>;
-};
+// const information = (title, subtitle) => (
+//   <div>
+//     <h1>{title}</h1>
+//     <h2>{subtitle}</h2>
+//   </div>
+// )
```
Comment (sumitd94) [inline components/Modal/index.js:9]: NIT: If its not needed, let's remove this code.

File: components/Modal/modal.module.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  background-color: whitesmoke;
+  width: 500px;
+  height: 300px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: .1em solid gray;
+  border-radius: .8em;
+  /* display: none; */
```
Comment (sumitd94) [inline components/Modal/modal.module.css:11]: NIT: If not needed, we can remove this.

File: pages/cart.js [hunk 3]
```diff
@@ -33,10 +34,22 @@ const Cart = (props) => {
   const [showSummary, setShowSummary] = useState(false);
   console.log({ showSummary });
```
Comment (sumitd94) [inline pages/cart.js:35]: No Console logs please.

File: pages/cart.js [hunk 4]
```diff
@@ -33,10 +34,22 @@ const Cart = (props) => {
...
+      fname="no coins"
+      lname={<Link href="/bank"> Go to Bank </Link>}
+    />;
+  };
+
   return (
     <div>
       <NavBar personData={personData} />
       <Header comp={<Link href="/shop">Go to Shop</Link>} />
+      <Modal tog={props.totalCartCost ? successMsg : failMsg} />
```
Comment (sumitd94) [inline pages/cart.js:52]: How about destructuring `props` ?

File: pages/checkout.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+            transform: translate(-50%, -50%);
+            border: 0.1em solid gray;
+            border-radius: 0.8em;
+          }
+          .modal div {
+            text-align: center;
+            margin-top: 5em;
+          }
+        `}
+      </style>
```
Comment (sumitd94) [inline pages/checkout.js:40]: I think this is not needed.
you already wrote the same css in file `components/Modal/modal.module.css` 
Import this css file here and use `className={classNames.modal}`

**Review by sumitd94 — CHANGES_REQUESTED** [review]

File: pages/checkout.js (hunk 5 above)
Comment (RohitKuwar) [reply pages/checkout.js:40]: @sumitd94 Thnks for suggestions 🙌

**Pull request closed without merging**